src/litserve/loops.py

```diff
@@ -228,33 +546,27 @@ def run_streaming_loop(lit_api: LitAPI, lit_spec: LitSpec, request_queue: Queue,
                 lit_api.decode_request,
```

Comment:
`decode_request` is also moved to preprocess loop, should this not only include `lit_api.preprocess` method?